### ISSUE TITLE
How come we are requiring such an old node version?

### ISSUE THREAD

**Issue by o-liver** [SAP-samples/sap-iot-samples#11]
When trying to install the bootstrap scripts I get the messages:
> npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'sap-iot-bootstrap@1.0.0',
npm WARN EBADENGINE   required: { node: '^12.0.0' },
npm WARN EBADENGINE   current: { node: 'v15.11.0', npm: '7.6.0' }
npm WARN EBADENGINE }
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@sap/xsenv@2.2.0',
npm WARN EBADENGINE   required: { node: '^6.0.0 || ^8.0.0 || ^10.0.0 || ^12.0.0' },
npm WARN EBADENGINE   current: { node: 'v15.11.0', npm: '7.6.0' }
npm WARN EBADENGINE }

IMO we should update to a [non-deprecated node version](https://nodejs.org/en/about/releases/)!

https://github.com/SAP-samples/sap-iot-samples/blob/02400016bc2d006ecf53e8999553b2538ba8e0f5/iot-bootstrap-scripts/package.json#L11

**Comment by marcuscbehrens** [comment]
agreed, we are working to update it to a new version as we speak ...
